daemon/cmd/status_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016-2020 Authors of Cilium
```

Comment:
I'll drop this change before merging, I just don't want to re-push again while the CI is running.